server/app/views/applicant/NorthStarApplicantIneligibleView.java

```diff
@@ -77,15 +77,17 @@ public String render(Params params) {
 
     // Manually construct a hyperlink with a runtime href and localized string. The hyperlink will
     // be inserted into another localized string in the Thymeleaf template.
-    String linkHref =
-        program.externalLink().isEmpty()
-            ? applicantRoutes.show(params.profile(), params.applicantId(), program.id()).url()
-            : program.externalLink();
-    String linkText =
-        params.messages().at(MessageKey.LINK_PROGRAM_DETAILS.getKeyName()).toLowerCase(Locale.ROOT);
-    String linkHtml =
-        "<a href=\"" + linkHref + "\" target=\"_blank\" class=\"usa-link\">" + linkText + "</a>";
-    context.setVariable("programLinkHtml", linkHtml);
+    String linkHref = program.externalLink();
```

Comment:
suggestion (blocking): leave this as is for now with a TODO. i will be building off of this in the north start work i'm about to be doing.

Reply:
Is it an option to leave this as is for now with a TODO? Same question for the other north star changes. Since the public can't access these views yet, I'm hoping leaving them as is won't affect anything.

Reply:
No biggie if not, I can add it back in when I get there.